content/en/users/cloud-compute/ec3/apps/_index.md

```diff
@@ -33,26 +33,54 @@ Check the
 To deploy a virtual cluster with
 [Marathon](https://mesosphere.github.io/marathon/),
 [Mesos](http://mesos.apache.org/), and
-[Chronos](https://mesos.github.io/chronos/) as an orchestration,
-please select Mesos + Marathon + Chronos from the list of available LRMS.
+[Chronos](https://mesos.github.io/chronos/) as an orchestration, please select
+Mesos + Marathon + Chronos from the list of available LRMS.
 
 ### OSCAR cluster
 
 To deploy
 [Serverless computing for data-processing applications](https://www.egi.eu/about/newsletters/serverless-computing-for-data-processing-applications-in-egi/)
 in EGI, please select OSCAR from the list of LRMS (Local Resource Management
-System). Use cases of applications that use the OSCAR framework for
-event-driven high-throughput processing of files:
-
-* [Plants Classification](https://github.com/indigo-dc/plant-classification-theano),
+System). OSCAR supports data-driven serverless computing for file-processing
+applications. A file upload, to the object storage backend [MinIO](http://minio.io),
+will trigger the execution of a chosen shell script running inside a user-defined
```

Comment:
maybe just a nitpick, only shell scripts can be run?

Reply:
Yes, it must be a shell script (than can execute anything that is inside the container image)